libraries/plugins/json_rpc/json_rpc_plugin.cpp

```diff
@@ -302,16 +322,18 @@ void json_rpc_plugin_impl::process_params( string method, const fc::variant_obje
                if( ( method == "call" && request.contains( "params" ) ) || method != "call" )
                {
                   fc::variant func_args;
-
                   try
                   {
-                     process_params( method, request, api_name, method_name, func_args );
+                     process_params(method, request, api_name, method_name, func_args, network_name);
                   }
                   catch( fc::assert_exception& e )
                   {
                      response.error = json_rpc_error( JSON_RPC_PARSE_PARAMS_ERROR, e.to_string(), fc::variant( *(e.dynamic_copy_exception()) ) );
                   }
 
+                  if( !network_name.size() )
+                     network_name = _default_network;
```

Comment:
might set network to _default_network in process_params(or at the top of this function) so there is no need for this check anymore

Reply:
Right, will rework